src/Distribution.idr

```diff
@@ -88,4 +88,5 @@ ClosedFormDistribution [1] Gaussian where
 
      in (expEach exponent) / denominator
 
+  ||| **NOTE** This function is not yet implemented.
```

Comment:
there are other non-trivial functions such as lbfgs and grid search that aren't implemented. Could leave this for v0.0.5?

Reply:
there are actually only the optimizers, so I will implement this, and either mark those as unimplemented or implement them